lib/src/dataflow/tuple.dart

```diff
@@ -70,6 +72,31 @@ class Aes {
 
   /// The represent point of [position] points.
   Offset get representPoint => shape.representPoint(position);
+
+  bool operator ==(Object other) =>
+      other is Aes &&
+      index == other.index &&
+      deepCollectionEquals(position, other.position) &&
+      shape == other.shape &&
+      color == other.color &&
+      gradient == other.gradient &&
+      elevation == other.elevation &&
+      label == other.label &&
+      size == other.size;
+
+  @override
+  String toString() {
```

Comment:
Added so that failing tests can format `Aes`es and show a more user-friendly error message